### PR TITLE
Feat/314add group

### LINKED ISSUE #314: Create a section to easily copy the group id
This would be similar to what appears when the api key is enabled, and you can easily copy it by clicking a button.

### PR DESCRIPTION
**Pull request by aguzmant103**
## Description

This feature fixes #314  by adding a "Copy group ID" button that copies the group ID.

## Does this introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

**Comment by aguzmant103** [comment]
@vplasencia lmk what you think with this change now

**Review by vplasencia — CHANGES_REQUESTED** [review]

File: apps/dashboard/src/pages/group.tsx [hunk 1]
```diff
@@ -48,6 +48,9 @@ export default function GroupPage(): JSX.Element {
     const { groupId, groupType } = useParams()
     const [_group, setGroup] = useState<Group | null>()
     const { hasCopied, setValue: setApiKey, onCopy } = useClipboard("")
+    const { hasCopied: hasCopied2, onCopy: onCopy2 } = useClipboard(
```
[the remark this answers is not in the document]
Comment (aguzmant103) [reply apps/dashboard/src/pages/group.tsx:51]: Oh yeah much more readable 😅

File: apps/dashboard/src/pages/group.tsx [hunk 2]
```diff
@@ -48,6 +48,9 @@ export default function GroupPage(): JSX.Element {
...
+        groupId as string
```
[the remark this answers is not in the document]
Comment (aguzmant103) [reply apps/dashboard/src/pages/group.tsx:52]: @vplasencia didn't think of that, much safer!

**Review by vplasencia — APPROVED** [review]
Great work 👌

**Pull request merged**